supply_energy.md

```diff
@@ -0,0 +1,291 @@
+
+#### Renewable Resources 
+
+GCAM's renewable resources include wind, solar, geothermal, hydropower, and biomass; some regions are also assigned a "traditional biomass" resource. In contrast to the depletable resources, whose cumulative stocks are explicitly tracked, renewable resource quantities in GCAM are always indicated in terms of annual flows. Wind, solar, geothermal, and hydropower are considered only as options for producing electricity, and are not traded between regions. Traditional biomass is only used by the buildings sector in selected regions. Of the six renewable energy resources, only biomass is (a) traded globally, and (b) used as an energy form or feedstock in a wide variety of sectors.
+
+In general, the costs of producing electricity from renewable energy forms consist of the sum of the resource costs described here, the technology costs, and in some cases, backup-related costs. The latter two components to the costs are documented in the [electricity sector](supply_energy.html#electricity).
+
+##### Wind
+
+All regions are assigned a wind energy supply curve, where the quantity is in exajoules (EJ) of electricity produced per year, and the price is in 1975$ per gigajoule (GJ) of electricity produced. Unlike fossil resources, uranium, or biomass, the quantities of wind energy are considered to be within "regional" markets; they can not be traded between the different modeled geopolitical regions. The [supply curves](details_energy.html#wind) in each region are derived from bottom-up analysis documented in [Zhou et al. (2012)](supply_energy.html#zhou2012). Note that in this supply curve formulation, the price is zero when the quantity is zero; this is because the wind resource supply curves only represent costs that increase with deployment due to the nature of the resources; these include factors such as increased transmission distances, reduced capacity factors, and more costly access to sites, among others. The remainder of the costs of wind electricity generation are in the technology and backup, described in the [electricity sector](supply_energy.html#electricity).
+
+##### Solar
+
+Solar energy is modeled as two separate resources: "global solar resource" and "distributed_PV", where the latter refers only to photovoltaic installations on residential and commercial buildings. As with wind, both of these resources are indicated in terms of electricity production. Global solar resource is modeled as an unlimited resource, and with a very low price; unlike with wind, it is assumed that marginal resource-related costs do not escalate with deployment levels. The technology and backup-related costs are documented in the [electricity sector](#electricity). Capacity factors and costs vary by region and type of solar (e.g., distributed PV, CSP). See [solar](details_energy.html#solar) for more details.
```

Comment:
- Should we add a sentence on "distributed_PV" mentioning data sources, the fact that it's not unlimited, etc.?
- "See [solar](details_energy.html#solar) for more details."  I think this should be deleted; it seems to be a reference to this section (there is not another "solar" section).

Reply:
Can you give me a sentence for the first bullet? I've made the change in the second.

Reply:
The info I was anticipating is actually in [solar] (details_energy.html#solar), which you reference.  Maybe, to avoid not mentioning "distributed_PV" after the first sentence, just add the summarized version below after the sentence that ends in "do not escalate with deployment levels":

"The "distributed_PV" supply curve is of the same functional form as the wind supply curve, with an upward-sloping function designed to capture the increases in costs with deployment. Resource potential in each region is based on estimated building floorspace."